terraform/modules/function/main.tf

```diff
@@ -21,6 +21,10 @@ data "aws_iam_role" "admin" {
   name = var.app == "dpc" ? "ct-ado-bcda-application-admin" : "ct-ado-${var.app}-application-admin"
 }
 
+data "aws_iam_role" "dasg_admin" {
+  name = "ct-ado-dasg-application-admin"
+}
+
```

Comment:
Where is this role found? We use ct-ado-bcda-application-admin in the greenfield accounts.

Reply:
Sure. It's for federal staff on DASG–I'm not part of the `ct-ado-bcda-application-admin` 😉

Reply:
Okay then can we add the bcda admin as well? It only serves to aid in troubleshooting and doesn't serve us well if we can't assume the role.

Reply:
Yes, exactly! That's why I've added the DASG admin here. The content of `data.aws_iam_role.admin.arn` defined just above will need to be adjusted for greenfield in a subsequent change set ahead of following through for `ab2d`, but `ct-ado-bcda-application-admin` is already there for gf-dev `dpc` and `bcda` resources.